src/components/PurchaseOrder/PO.js

```diff
@@ -124,6 +131,26 @@ class PO extends Component {
     return false;
   }
 
+  closeOrder = (values) => {
+    const { mutator, resources, onCancel } = this.props;
+    const initialValues = get(resources, ['order', 'records', 0]);
+    const deep = cloneDeep(initialValues);
+
+    delete deep.created_by_name;
+    delete deep.assigned_to_user;
+    delete deep.vendor_name;
+    delete deep.po_lines;
+    mutator.order.PUT({
+      ...deep,
```

Comment:
if you make a deep clone, no need to create new object here - you already have one, `deep`. Also, `deep` - is a bad name, doesn't reflect what contains a variable